File: C/C-Sheet #2-Loop/N.c
#include <stdio.h>

int main() {
  char opr;
  scanf("%c", &opr);
  int lines;
  scanf("%d", &lines);
  for (int i = 1 ; i <= lines ;i++){
      int x;
      scanf("%d", &x);
      while (x--){
        printf("%c", opr);
      }
        printf("\n");
  }

  return 0;
}

File: Problem Solving/CodeForces/ICPC Assiut University Training Sheet/Newcomers/Java/C-Sheet #2-Loop/N.java
import java.util.Scanner;

public class N {
  public static void main(String[] args){
  Scanner input = new Scanner(System.in);
  String opr;
  opr = input.next();
  int lines;
  lines = input.nextInt();
  for (int i = 1 ; i <= lines ;i++){
      int x;
      x = input.nextInt();
      for(int j=1;j<=x;j++){
        System.out.print(opr);
      }
      System.out.println();
  }
  input.close();
  }
}